domains/polar.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "polarfreez",
+        "email": "rafaelbraun5@hotmail.com"
+    },
+    "record": {
+        "CNAME": "https://polarfreez.github.io"
```

Comment:
CNAME cannot have https:// (also site gives 404)

Reply:
Fixed. Can you please check again?